internal/buildengine/deploy_integration_test.go

```diff
@@ -0,0 +1,23 @@
+//go:build integration
+
+package buildengine
+
+import (
+	"testing"
+
+	in "github.com/TBD54566975/ftl/internal/integration"
+)
+
+func TestDeploy(t *testing.T) {
+	in.Run(t,
+		in.WithTestDataDir("testdata"),
```

Comment:
We can remove this once all tests are integrated, and we can move the modules under corresponding language dirs.